src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -100,10 +103,16 @@ const getMpuComponent =
     allowAdvertising ? <AdContainer {...props} slotType="mpu" /> : null;
 
 const getWsojComponent =
-  (recommendationsData: Recommendation[]) =>
-  (props: ComponentToRenderProps) => (
-    <CpsRecommendations {...props} items={recommendationsData} />
-  );
+  (service: Services) =>
+  (props: ComponentToRenderProps & { data: Recommendation[] }) => {
+    // TODO: Remove this when the new recommendations are rolled out to all services
+    if (SERVICES_WITH_NEW_RECOMMENDATIONS.includes(service)) {
```

Comment:
As above - can we use a toggle to control the release instead?

Reply:
Same as above, don't think its particularly worth it as we'll just be deleting this logic, and then ultimately the toggle if its there. Its just temporary until editorial communicate the change.